ikasaneip/builder/jar/src/main/java/org/ikasan/builder/MonitorBuilder.java

```diff
@@ -41,13 +41,12 @@
 package org.ikasan.builder;
 
 import org.ikasan.monitor.MonitorFactory;
-import org.ikasan.monitor.notifier.DashboardNotifier;
 import org.ikasan.monitor.notifier.EmailNotifierConfiguration;
 import org.ikasan.monitor.notifier.NotifierFactory;
 import org.ikasan.spec.configuration.Configured;
 import org.ikasan.spec.configuration.ConfiguredResource;
-import org.ikasan.spec.configuration.PlatformConfigurationService;
 import org.ikasan.spec.dashboard.DashboardRestService;
+import org.ikasan.spec.monitor.FlowMonitor;
 import org.ikasan.spec.monitor.Monitor;
 import org.ikasan.spec.monitor.Notifier;
 
```

Comment:
Shouldnt the calss be called FlowMonitorFactory

Reply:
@andrzej-majewski i am not sure which class you mean?